app/bot/openai.go

```diff
@@ -18,38 +19,73 @@ type OpenAIClient interface {
 	CreateChatCompletion(context.Context, openai.ChatCompletionRequest) (openai.ChatCompletionResponse, error)
 }
 
+// OpenAIConfig contains parameters for OpenAI bot
+type OpenAIConfig struct {
+	AuthToken       string
+	MaxTokens       int
+	Prompt          string
+	HistorySize     int
+	HistoryRandBase int64
```

Comment:
this one is confusing a little bit. I get what it means (1/base probability) but in a swe weeks I'll forget it completely. My suggestion is to rename it to something mor obvious like HistReplyProbability and add a comment saying // 0 -never reply, 1 replay to each message, 0.2 replay to 20% of messages